ReadModel/src/ModelGenerator.cs

```diff
@@ -0,0 +1,70 @@
+﻿using EventReader.Event;
+using EventReader.Read;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ReadModel
+{
+    public class ModelGenerator
+    {
+        EventStream Stream;
+        public Dictionary<Guid, Customer> Customers { get; }
+
+        const string CUSTOMER_QUERY =
```

Comment:
Ahh right, I'll fix that. I was thinking that it would be some parameter passed from the top down, depending on what the user wanted, but that makes sense. I'll make it constant closer to source. Thanks.